jenkins_jobs/trigger_yamllint_from_versions_repo/script.sh

```diff
@@ -0,0 +1,2 @@
+YAML_FILES=$(find . -name "*.yaml")
+yamllint $YAML_FILES
```

Comment:
Shouldn't we be executing the validate_yamls.py script here?
I've also tested and this command failed because it's not reading the .yamllint file.
The same applies for rpmlint.